### ISSUE TITLE
Sliding to a previous page shows black screen for a moment

### ISSUE THREAD

**Issue by DavisJ3608** [laurencedawson/sync-for-lemmy#329]
**Describe the bug**  
When sliding back to a previous screen, like going back to the home screen after viewing an individual post, the screen displayed "below" that pane is completely black or, on rare occasions, the phone's wallpaper. The intended page will then display after a moment.

**Steps to Reproduce**  
1) Navigate to a new page
2) Use the "slide right" action to navigate to a previous page

**Expected behavior**  
The previous page has always displayed "underneath" the page that's being navigated away from.

**Screenshots**  
https://github.com/laurencedawson/sync-for-lemmy/assets/5335681/5202db17-891a-4a52-adf8-83aed89b1fa3

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy S23 Ultra
 - Android: 14 (One UI 6.0 Beta)
 - Sync version: v23.08.12-23:05 (29)
 - Sync variant: Google Play

**Comment by laurencedawson** [comment]
Does this happen every time?

**Comment by corylowry12** [comment]
Happens to me sometimes as well, am unable to make it happen though

**Comment by DavisJ3608** [comment]
> Does this happen every time?

Yes, 100% of the time. May be related to the One UI beta, I'm also getting a bunch of weird UI glitches for things like going to multi-window.

**Comment by DavisJ3608** [comment]
Just wanted to provide an update on this - just updated to the latest Samsung beta software which has resolved all (or at least all that I've noticed) weird UI bugs throughout the OS. This issue still persists - 100% of the time, navigating back shows either a black screen or the home screen before updating to the previous page.

**Comment by ciotorcristian** [comment]
I have also updated on OneUI 6 on my GalaxyS21 and this bug appeared on the old Sync for Reddit app and also the new Sync for lemmy.
